apps/src/code-studio/pd/professional_learning_landing/LandingPage.jsx

```diff
@@ -38,9 +38,12 @@ export default class LandingPage extends Component {
           )}
           <EnrolledWorkshops />
           {this.props.deeperLearningCourseData && (
-            <ProfessionalLearningCourseProgress
-              deeperLearningCourseData={this.props.deeperLearningCourseData}
-            />
+            <div>
+              <h2>Online Professional Learning Courses</h2>
```

Comment:
This wasn't translated before and is temporary, so I didn't bother to translate it now.